evolvedbinary/jdita
# Documentation for JDita

This PR contains the documentation of the JDita project.
All functions, classes, interfaces, and variables have been documented with comment headers.
The Readme has been updated accordingly.
Additionally, we have added 3 diagrams to visualise the application flow.

---

**How to test this PR:**

* Before you test to build this project, make sure to remove the `node_modules` folder from previous builds & branches.
* Run `yarn install`
* Run `yarn run generate-docs`
* Check the generated docs in folder `./generated-docs`
